.zappr.yaml

```diff
@@ -1,15 +1,20 @@
 approvals:
-  # PR needs at least 1 approval
-  minimum: 1
+  # PR needs at least 2 approvals
+  minimum: 2
   # approval = comment that matches this regex
   pattern: "^(:\\+1:|👍)$"
   # note that `from` is by default empty,
   # accepting any matching comment as approval
-  from:
-    # commenter must be either one of:
-    orgs:
-      # a member of norris organization
-      # (any org in here counts)
+  groups:
+   # there must be at least a technical approval from a member of team Norris
+   technical:
+    minimum: 1
+    from:
+     orgs:
       - norris
```

Comment:
One other thing... I don't see the zappr structure that you've included in any of the following examples:

* https://github.com/zalando/zappr/blob/master/config/app-opensource.yaml
* https://github.com/zalando/zappr/blob/master/config/app-enterprise.yaml
* https://gist.github.bus.zalan.do/tyang/37575c0177beb06cdab4ced60e4bd389

How can you tell that the zappr structure you've added is correct?